### PR TITLE
CIV-4407 Not authorised to view the claim

### PR DESCRIPTION
**Pull request by hmcts-version1-domini**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-4407

### Change description ###

Develop 'You are not authorised to view the claim!' screen: /first-contact/access-denied

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by TatsyMaslova** [comment]
I am not sure whether this is complete. the idea is that we show this page every time we have 401 Not Authorise. similar to when we show 404 page not found and similar to error page. So the idea is that we show this page from whenever. And I don't see this in here. also there won't be authentication because just like in 404 and error page we don't need authentication to show this and we don't need a controller for this page.

**Comment by hmcts-version1-domini** [comment]
> 

This is the first AC though:

Given citizen has input security code
And The PIN is incorrect
And has the right format 
When 'Continue' is selected
Then display 'You are not authorised to view the claim!' screen (first-contact/access-denied)

**Comment by TatsyMaslova** [comment]
> 

Yes, but during refinement we discussed that we are going to ensure that this page is shown whenever we have 401 so there is no logic in the controller that it will redirect or display the view. similar to how we do with errors. we story pointed it according to what we discussed and agreed with Sanpan

File: src/test/unit/views/features/public/firstContact/access-denied.test.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    it('should have correct page title', () => {
+      expect(htmlDocument.title).toEqual('Your money claims account - You are not authorised to view the claim!');
+    });
+
+    it('should have correct page title', () => {
+      const hyperlink = htmlDocument.getElementById('claimReferenceUrl');
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      expect(hyperlink.getAttribute('href')).toContain('/first-contact/claim-reference');
```
[the remark this answers is not in the document]
Comment (hmcts-version1-domini) [reply src/test/unit/views/features/public/firstContact/access-denied.test.ts]: Yes, this was the reason. Ok, already changed with this operator. Thanks

**Review by TatsyMaslova — CHANGES_REQUESTED** [review]
as I have put comment above we agreed that this page would show on any 401 indecently whether it comes from the page in acceptance criteria.

**Pull request merged**